src/cdk-experimental/menu/menu-stack.ts

```diff
@@ -55,15 +55,43 @@ export class MenuStack {
   }
 
   /**
-   *  Pop off the top most MenuStackItem and emit it on the close observable.
+   *  Pop off the top most MenuStackItem and emit it on the close observable if the stack is not
+   *  empty. If `lastElement` is provided it will pop off until this element (inclusive). If
+   * `lastElement` is  not in the stack, it will pop off the topmost element only.
```

Comment:
If you're referring to what happens when `lastElement` is not provided that was just a fallback behaviour decision for which under normal circumstances it shouldn't be encountered. I figured that in the edge case where that may occur it would be better to pop off rather than do nothing. I'm flexible to either approach though.